easybuild/easyconfigs/j/JUBE/JUBE-2.0.3.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = "VersionIndependentPythonPackage"
+
+name = "JUBE"
+version = "2.0.3"
+
+homepage = "http://www.fz-juelich.de/jsc/jube"
+description = """The JUBE benchmarking environment provides a script based framework to easily create benchmark sets,
+ run those sets on different computer systems and evaluate the results."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ["http://apps.fz-juelich.de/jsc/jube/jube2/download.php?version=%(version)s"]
```

Comment:
there was an issue here, this needs to be changed to make sure the right version is downloaded...